spec/openapi.yaml

```diff
@@ -237,6 +237,39 @@ tags:
       [APIAuth](https://github.com/mgomes/api_auth) Ruby Gem 
 
       ## Postman Collections
+      
+  - name: Sub-accounts
+    description: |
+      ## Performing actions on behalf of sub-accounts
+      Using API keys at the parent account level, you can perform actions on behalf of a sub-account. 
+      
+      This feature is supported by the Messages, Replies, Delivery Reports and Webhooks APIs. 
+      
+      To do this
+        include a header key ```Account``` with the sub-account ID as the value. For example:
+        ```Account: mySubAccount```
+
+        **Example request with Sending from sub-accounts**
+      
+        ```plain
+        POST /v1/messages HTTP/1.1
+        Host: api.messagemedia.com
+        Accept: application/json
+        Content-Type: application/json
+        Authorization: Basic dGhpc2lzYWtleTp0aGlzaXNhc2VjcmV0Zm9ybW1iYXNpY2F1dGhyZXN0YXBp
+        Account: SubAccount
+        {
+          "messages": [
+            {
+              "content": "Hello World",
+              "destination_number": "+61491570156",
+              “delivery_report”: true
+            }
+          ]
+        } 
+        ```
+        *Note: when sending multiple messages in a request, all messages must be valid for the request 
```

Comment:
you can remove the "Note" from this section, just needs to be in the Messages section below :)